pyfr/readers/base.py

```diff
@@ -22,3 +26,195 @@ def to_pyfrm(self):
         mesh['mesh_uuid'] = np.array(str(uuid.uuid4()), dtype='S')
 
         return mesh
+
+
+class MeshDBAssembler(object):
```

Comment:
Sorry for the poor name of this class. Could you please suggest a good name for this?